scss/samples/_mixins-functions.scss

```diff
@@ -7,7 +7,32 @@
 /// [SassDoc annotations](http://sassdoc.com/annotations/).
 /// This page includes samples of every core annotation type,
 /// so you can see what they look like in Herman.
+/// @group demo_test-sassdoc
+
+/// Extra Commentary
+/// ================
+/// Herman allows you to write "prose" markdown blocks,
+/// to help introduce groups,
+/// or add narrative flow between documented items.
 ///
+/// Note that while SassDoc allows annotation comments
+/// to be separated from the documented code by newlines,
+/// Herman considers documentation to be free-floating "prose" if
+/// it is separated from documented code by one or more newlines.
+///
+/// This change often breaks the `autofill` functionality
+/// of the core SassDoc annotations, so **Herman disables all
+/// annotation `autofill` functionality by default** since v5.0.0.
+/// To allow autofilled annotations, set the `HERMAN_ENABLE_AUTOFILL`
+/// environment variable to any value before running SassDoc.
+///
+/// Prose blocks can also include examples, and other annotations.
+/// @example scss
+///   .private::after {
+///     content: samples.$private;
+///   }
+/// @link http://sassdoc.com/annotations/
+/// @link http://sassdoc.com/configuration/#autofill
```

Comment:
These examples aren't very clear in the rendered version. I got to them and wondered if I was seeing a bug. I think we would need to show an escaped version of the input, and then the rendered annotations would make more sense. I guess that's not new, though - just moved.

Reply:
@mirisuzanne Addressed in https://github.com/oddbird/sassdoc-theme-herman/pull/376/commits/b05d6dfca0a5ca42d84f13595eea8a5f25c2f6f2. What do you think?